tests/test_mlkit.py

```diff
@@ -299,7 +363,7 @@ def test_model_format_setters(self):
     def test_model_display_name_validation_errors(self, display_name, exc_type):
         with pytest.raises(exc_type) as err:
             mlkit.Model(display_name=display_name)
-        check_error(err.value, exc_type)
+        check_error(err, exc_type)
```

Comment:
Lets rename err to excinfo to be consistent with other test modules and the helper functions above.

Reply:
done